rust/rust.dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM janx/ubuntu-dev
+MAINTAINER Jan Keromnes "janx@linux.com"
+
+RUN sudo apt-get update -q \
+ && sudo apt-get upgrade -qy \
+ && sudo DEBIAN_FRONTEND=noninteractive apt-get install -qy \
+  build-essential \
+  cmake
+
+# Download Rust's source code.
+RUN git clone https://github.com/rust-lang/rust
```

Comment:
Nit: I prefer to be painfully explicit about where this should be cloned to, i.e. `git clone https://github.com/rust-lang/rust /home/user/rust`.